components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -324,7 +324,6 @@ const VulnerabilitiesGrid = ({ resources, onUpdate, residentCoordinates, generic
                               {...resource}
                               updateSelectedResources = {updateSummaryResource}
                               customerId={customerId}
-                              foodNeedsId = {id}
```

Comment:
I don't understand this removal as part of this PR - I think I'm missing something?

Reply:
Originally (in the previous pr), I was doing the styling based on whether or not it was a food category. The assumption there was that summaries only existed for this category. That not correct, so now I'm checking if any of those summaries (telephone, distance etc )are included which is now what determined when the styling should be applied.

Reply:
Ah, OK, so this now isn't needed for this, I understand.